default/generated/azure/01-azure-aws-config.windup.yaml

```diff
@@ -14,7 +12,7 @@
   - konveyor.io/target=azure-container-apps
   - AWS
   links: []
-  message: The application contains AWS credential configuration.
+  message: "\n \n The application contains AWS credential configuration.\n \n "
```

Comment:
hmm, do we want these leading trailing newlines?  I feel like this could make our display look pretty weird

Reply:
They are a consequence of directly translating things like this:

```xml
<message>

Description for the rule, blablabla

</message>
```

They can easily be removed by sed'ing. I can do that in this PR or wait and create another one after this.

Reply:
I feel like we should strip it in the shim, shouldn't need to do it by hand. That can be a follow-up